class NaoEncontradoException extends Error {
  constructor (modelo = '') {
    super(`${modelo} n√£o encontrado`)
    this.nome = 'NaoEncontrado'
    this.idError = 2
  }
}

module.exports = NaoEncontradoException
